CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- Added `queryConditions` config variable on Sage People component
```

Comment:
Way to go remembering to update the changelog! 😁 💯

Reply:
Thanks